### PR TITLE
add support for elasticsearch logging

### PR DESCRIPTION
**Pull request by ssoper-usgs**
Before making a pull request
----------------------------

- [x] Make sure all tests run
- [ ] Update the changelog appropriately

Title
-----------
https://internal.cida.usgs.gov/jira/browse/IOW-726

Description
-----------
* Added support for AWS elasticsearch logging

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -27,11 +33,20 @@ custom:
     subnetIds: ${ssm:/iow/aws/vpc/${self:provider.stage}/subnetIds~split}
   db:
     connectInfo: ${ssm:/aws/reference/secretsmanager/NWCAPTURE-DB-${self:provider.stage}~true}
+  esLogs:
+    endpoint: ${ssm:/iow/aws/vpc/${self:provider.stage}/elasticsearchEndpoint}
+    index: ${self:service}
+    retentionInDays: 90
+    filterPattern: '?Exception ?ERROR ?WARN'
```
Comment (kkehl-usgs) [inline serverless.yml:40]: I thought it was ?ERROR ?CRITICAL elsewhere ...

**Review by kkehl-usgs — APPROVED** [review]

File: serverless.yml (hunk 1 above)
Comment (ssoper-usgs) [reply serverless.yml:40]: That's a good point... I erred for WARN and up.  Should this actually be something like `'?Exception ?ERROR ?WARN ?FATAL'?

Comment (kkehl-usgs) [reply serverless.yml:40]: Maybe it was a python vs java thing.  I don't know what the options are, I just saw a (potential) inconsistency if some things are passing error + critical and some are passing error + warn.

Comment (ayan-usgs) [reply serverless.yml:40]: `?ERROR ?CRITICAL` is being used for the python lambdas.

**Review by ayan-usgs — APPROVED** [review]

**Pull request merged**